tests/rules/no-internal.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license Use of this source code is governed by an MIT-style license that
+ * can be found in the LICENSE file at https://github.com/cartant/eslint-plugin-rxjs
+ */
+
+import { stripIndent } from "common-tags";
+import rule = require("../../source/rules/no-internal");
+import { ruleTester } from "../utils";
+
+ruleTester({ types: false }).run("no-internal", rule, {
+  valid: [
+    `import { Observable } from "rxjs";`,
+    `import { map } from "rxjs/operators";`
+  ],
+  invalid: [
+    {
+      code: stripIndent`
+        import { concat } from "rxjs/internal/observable/concat";
+        import { map } from "rxjs/internal/operators/map";`,
+      errors: [
+        {
+          messageId: "forbidden",
+          line: 1,
+          column: 24,
+          endLine: 1,
+          endColumn: 57
+        },
+        {
+          messageId: "forbidden",
+          line: 2,
+          column: 21,
+          endLine: 2,
+          endColumn: 50
+        }
+      ]
+    },
+    {
+      code: stripIndent`
+        import { concat } from 'rxjs/internal/observable/concat';
+        import { map } from 'rxjs/internal/operators/map';`,
```

Comment:
Isn't this test identical to the one above? Did you intend to split the the two imports into two tests, so that each test includes only a single import?

Reply:
One for single quotes and the other one for double quotes.
In the tslint code the selector checked for quotes because it used the "raw" value of the import. We know check on the value (so without any quotes) so, this can indeed be removed 👍

Reply:
👍🙈